easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.2.0-intel-2017a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenEXR'
+version = '2.2.0'
+
+homepage = 'http://www.openexr.com/'
+description = """OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic
+ for use in computer imaging applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/openexr/openexr/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['8c219754af33fee199bdd72a0441f73dd4b3ce5363ac3c5cd7b2c6306f5442a6']
+
+builddependencies = [('CMake', '3.9.1')]
+
+env_vars = "CPATH=%(installdir)s/include/OpenEXR:$CPATH "
+env_vars += "LIBRARY_PATH=%(installdir)s/lib:$LIBRARY_PATH "
+env_vars += "LD_LIBRARY_PATH=%(installdir)s/lib:$LD_LIBRARY_PATH"
```

Comment:
this needs the install path already included? what happens if you do a rebuild?

Reply:
Rebuild works fine, just tested that.

Without this included it fails during the build step, you get errors like:

```
catastrophic error: cannot open source file "half.h"
```

EasyBuild performs two configure-build-install cycles (see list value for `preconfigopts`), and the `IlmBase` stuff must be available already when installing the `OpenEXR` stuff; when `IlmBase` is installed, the install directory is wiped if it existed already, and only when `OpenEXR` is being built are these env var updates put in place.